pjsip/src/pjsua2/account.cpp

```diff
@@ -117,6 +119,13 @@ void RtcpFbConfig::writeObject(ContainerNode &node) const PJSUA2_THROW(Error)
 
 ///////////////////////////////////////////////////////////////////////////////
 
+SendRequestParam::SendRequestParam()
+: method("")
```

Comment:
Minor: better initialize `token` (to NULL) too, some code analyzer usually complains :)